docs/shared/getting-started/intro.md

```diff
@@ -1,118 +1,85 @@
 ---
 
-# Intro to Nx
+# Introduction
 
-Nx is a powerful open-source build system that provides tools and techniques for enhancing developer productivity, optimizing CI performance, and maintaining code quality. [Check out our video](/getting-started/why-nx) to learn more about what Nx is about.
+Welcome to the official Nx documentation!
 
-{% github-repository title="Star Nx on GitHub" url="https://github.com/nrwl/nx" /%}
+## What is Nx?
 
-## Core Features
+Nx is a powerful, open source, technology-agnostic build platform designed to efficiently manage codebases of any scale. From small single projects to large enterprise monorepos, Nx is designed to **streamline your development workflows**, **reduce errors** and **dramatically reduce waste** by [saving engineering time](https://youtu.be/2BLqiNnBPuU) and cutting compute costs.
 
-- **Run Tasks Efficiently**: Nx [runs tasks in parallel](/features/run-tasks) and orders the tasks based on the dependencies between them.
-- **Distribute Tasks in CI**: Nx scales your CI by [distributing tasks](/ci/features/distribute-task-execution) across many VMs. Your CI is fast no matter how big your repository is.
-- **Cache Locally & Remotely**: With [local](/features/cache-task-results) and [remote caching](/ci/features/remote-cache), Nx prevents unnecessary re-runs of tasks, saving you valuable dev time.
-- **Split E2E Tests and Rerun Flaky Tests**: Nx [automatically splits](/ci/features/split-e2e-tasks) large e2e tests to distribute them across VMs. Nx can also automatically [identify and rerun flaky e2e tests](/ci/features/flaky-tasks).
-- **Automate Dependency Updates**: if you leverage [Nx plugins](/concepts/nx-plugins) you gain additional features such as [code generation](/features/generate-code) and tools to [automatically upgrade](features/automate-updating-dependencies) your codebase and dependencies.
-- **Make it Your Own**: Nx is highly customizable and extensible. Fine-tune it by [creating a plugin for your organization](/extending-nx/tutorials/organization-specific-plugin) or [creating a tooling plugin](/extending-nx/tutorials/tooling-plugin).
+{% youtube
+src="https://www.youtube.com/embed/-_4WMl-Fn0w"
+title="What is Nx?"
+width="100%" /%}
 
-<!-- - **Monorepo and Single Projects**: Nx supports both, monorepos as well as single-project (standalone) workspaces. -->
+## How Nx solves scaling
 
-Find out more about [why you should use Nx](/getting-started/why-nx) or browse our [features](/features).
+Monorepos offer significant advantages. They reduce integration overhead between different software components, enabling faster cross-project changes, and improving code sharing and collaboration among teams. However, without proper tooling, as projects multiply, build times increase and management complexity grows, potentially negating the original benefits.
```

Comment:
```suggestion
Monorepos offer significant advantages. They reduce integration overhead between different software components, enabling faster cross-project changes, and improving code sharing and collaboration among teams. However, when projects multiply, build times increase and management complexity grows, potentially negating the original benefits.
```

No need to clarify without proper tooling due to you saying in the next sentence Nx solves this problem. IMO.